### PR TITLE
Fixed Maintenance Banner

### PR DESCRIPTION
**Pull request by ianmagenta**
## Summary
Updated the `DowntimeBanner` component so that it displays when it's unable to get an OK response from the `/backend_statuses` endpoint.

## Related issue(s)
https://github.com/department-of-veterans-affairs/va.gov-team/issues/73465

## What areas of the site does it impact?
This impacts the `DowntimeBanner` component which is displayed on both the sign-in modal and the sign-in page.

## Acceptance criteria
- [x] Investigate which maintenance banner the identity team uses
- [x] Update vets-website so that a 400/500 or other error codes returned from `/backend_statuses` displays the maintenance banner
- [x] Update unit tests as needed

### CODE REVIEW

**Comment by ianmagenta** [comment]
@asg5704 - All of the `DowntimeBanner` tests are currently set to be skipped, so I didn't include any new ones for this change. If you want me to add one though, happy to do so!

File: src/platform/user/authentication/components/DowntimeBanner.js [hunk 1]
```diff
@@ -9,14 +9,19 @@ export default function DowntimeBanners() {
...
     }
   }, []); // only on load
 
-  if (!statuses) return null;
-  const bannerStatus = getStatusFromStatuses(statuses);
+  // mimics the ssoe service error if we don't get an OK response from vets-api
+  const statusArray =
+    statuses || isLocalhost
+      ? []
+      : [{ service: 'ssoe', serviceId: 'ssoe', status: 'down' }];
```
Comment (asg5704) [inline src/platform/user/authentication/components/DowntimeBanner.js:23]: Change this to `mvi` one in the constants

**Review by asg5704 — CHANGES_REQUESTED** [review]
Also see the Slack messages for more context

**Review by asg5704 — APPROVED** [review]
Looks good Ian. Thank you!

**Review by rsmithadhoc — APPROVED** [review]

**Pull request merged**